sw/airborne/modules/multi/traffic_info.c

```diff
@@ -79,7 +79,11 @@ bool parse_acinfo_dl(void)
   uint8_t msg_id = IdOfPprzMsg(dl_buffer);
 
   /* handle telemetry message */
+#if PPRZLINK_DEFAULT_VER == 2
```

Comment:
Looks good, only I just now noticed this PPRZLINK_DEFAULT_VER in the pprzlink repo. For my own info, is there a reason it is 'default', is there a way to have multiple versions simultaneously where one is default and another isn't? Why not just pprzlink_version?